app/controllers/spotlight/catalog_controller.rb

```diff
@@ -201,4 +201,13 @@ def current_page_context
       end
     end
   end
+
+  def additional_export_formats document, format
+    super
+
+    format.solr_json do
+      authorize! :update_solr, @exhibit
+      render json: document.to_solr.merge(@exhibit.solr_data)
```

Comment:
It'd be nice if you could get this data from a global endpoint, e.g.  `/catalog/x.solr_json`, but we don't have a list of all the exhibits a document should be in, so I guess we have to settle for this.